src/api/auth/src/authentication.js

```diff
@@ -107,17 +107,43 @@ const strategy = new SamlStrategy(
      *   "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress": "first.last@email.com"
      *   "email": "first.last@email.com"
      * }
+     *
+     * We only really care about the emailaddress claim, which we also use in the Users service.
      */
+    const email =
+      senecaProfile['http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress'];
+
+    async function lookupTelescopeUser(id) {
+      // We have the user's id (i.e. their email address) in a valid token. Now
+      // make a service-to-service request to the Users service in order to
+      // get this user's full profile information using the token's subject claim
+      // as the user's id (e.g., their email address).
+      try {
+        const res = await fetch(`${USERS_URL}/${id}`, {
+          headers: {
+            Authorization: `bearer ${createServiceToken()}`,
```

Comment:
We should probably add a Cache-Control header that contains `private` or `no-store` due to security risks on URI's/URL's

see:  https://tools.ietf.org/html/rfc6750#section-5